.github/DISCUSSION_TEMPLATE/question.yml

```
@@ -0,0 +1,62 @@
+title: "Summary"
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We are glad to have you build on Aptos with us!
+        Please fill out the questions below.
+  - type: textarea
+    id: question
+    attributes:
+      label: Describe your question in detail. For example how do you get all transactions for a user with the TypeScript SDK?
+    validations:
+      required: true
+  - type: textarea
+    id: error
+    attributes:
+      label: What error, if any, are you getting?
+      description: |
+        If no error, you can leave this field blank.
```

I screwed up the english here on my last pass
```suggestion
        If there is no error, you can leave this field blank.
```